src/psyclone/domain/gocean/nodes/gocean_extract_node.py

```diff
@@ -141,66 +142,111 @@ def create_driver(self, input_list, output_list):
                      funcnames=["PSyDataType"])
         prog.add(use)
 
+        # Use a symbol table to make sure all variable names are unique
+        sym_table = GOSymbolTable()
+        sym = Symbol("PSyDataType")
```

Comment:
You could use `sym_table.name_from_tag("PSyDataType")` here (and again below) as that adds the new symbol to the table too. However I note #720 that it is due to be deprecated. Do @sergisiso or @rupertford have any suggestions?

Reply:
I'll wait for an answer here :)

Reply:
OK, I had another look: and name_from_tag will return an existing symbol, which is not useful - I have to make sure I have a unique symbol. So I am just leaving this the way it is.

Reply:
I think `new_symbol_name()` is the right way to go.